frameworks/hdfs/tests/test_sanity.py

```diff
@@ -14,70 +11,62 @@
 import sdk_tasks
 import sdk_upgrade
 import sdk_utils
-from tests.config import (
-    DEFAULT_HDFS_TIMEOUT,
-    DEFAULT_TASK_COUNT,
-    FOLDERED_SERVICE_NAME,
-    PACKAGE_NAME,
-    ZK_SERVICE_PATH,
-    check_healthy,
-    expect_recovery,
-    get_pod_type_instances
-)
+import shakedown
+from tests import config
```

Comment:
The short answer is making calls to outside modules more explicit and easier to track in a consistent way. `dcos/dcos` is using the Google Python Style Guide (with a few local exceptions -- like 120 character line length), and it uses package- and module-level importing.